src/grimp/application/ports/importscanner.py

```diff
@@ -16,18 +16,21 @@ def __init__(
         file_system: AbstractFileSystem,
         found_packages: Set[FoundPackage],
         include_external_packages: bool = False,
+        include_type_checking_imports: bool = True,
```

Comment:
Could we change this to `exclude_type_checking_imports: bool = False` - and everywhere else? I'd like it to feel more like exceptional / unusual behaviour.

Also, non-blocking but I think this would be better as an argument passed to `scan_for_imports` rather than to `__init__`, as it makes the ImportScanner a little less stateful (we could use the same `ImportScanner` instead to scan in both ways).

Reply:
I used exclude to match the other parameters, but hesitated a lot. Will change

Reply:
Done